lib/ansible/modules/monitoring/datadog_monitor.py

```diff
@@ -29,6 +29,14 @@
           - Your Datadog API key.
         required: true
         type: str
+    api_host:
+        description:
+          - The URL to the Datadog API.
+        required: false
+        type: str
+        choices: ['https://api.datadoghq.com', 'https://api.datadoghq.eu']
```

Comment:
In case another endpoint is added, allowed values should not be constrained. Current known values could be mentioned in the description.